iroh-p2p/src/node.rs

```diff
@@ -542,6 +553,47 @@ impl<KeyStorage: Storage> Node<KeyStorage> {
                         QueryResult::Bootstrap(Err(e)) => {
                             warn!("kad bootstrap error: {:?}", e);
                         }
+                        QueryResult::GetClosestPeers(Ok(GetClosestPeersOk { key, peers })) => {
```

Comment:
this might be called multiple times, only if `step.last` is `true` is it the last one and you should abor/error out